spikeinterface/core/job_tools.py

```diff
@@ -34,8 +34,23 @@
                 Note that "fork" is only available on UNIX systems
     """
     
-job_keys = ['n_jobs', 'total_memory', 'chunk_size', 'chunk_memory', 'chunk_duration', 'progress_bar', 
-            'mp_context', 'verbose', 'max_threads_per_process']
+job_keys = ('n_jobs', 'total_memory', 'chunk_size', 'chunk_memory', 'chunk_duration', 'progress_bar', 
+            'mp_context', 'verbose', 'max_threads_per_process')
+
+
+def split_job_kwargs(mixed_kwargs):
+    """
+    This split mixed kwargs into job_kwargs and specific_kwargs.
+    This can be usefull for some function with generic signature
+    mixing specific and job kwargs.
```

Comment:
```suggestion
    This function splits mixed kwargs into job_kwargs and specific_kwargs.
    This can be useful for some function with generic signature
    mixing specific and job kwargs.
```